src/kalastatic.js

```diff
@@ -1,9 +1,10 @@
 'use strict'
-var assert = require('assert')
-var path = require('path')
-var kss = require('kss/lib/cli')
-var Metalsmith = require('metalsmith')
-var extend = require('extend')
+
+const assert = require('assert')
+const path = require('path')
+const kss = require('kss/lib/cli')
+const Metalsmith = require('metalsmith')
+const extend = require('extend')
```

Comment:
Should we switch to es6 imports?
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/import